framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -2286,40 +2285,13 @@ protected boolean checkMethodReferenceAsOverride(MemberReferenceTree overriderTr
         // ========= Overriding Executable =========
         // The ::method element
         ExecutableElement overridingElement = (ExecutableElement)InternalUtils.symbol(overriderTree);
+        AnnotatedExecutableType overrider =
```

Comment:
Why is it sometimes `overrider` and sometimes `overriding`? Can we make that more consistent?